cmd/mesh/upgrade.go

```diff
@@ -54,20 +55,24 @@ type upgradeArgs struct {
 	skipConfirmation bool
 	// force means directly applying the upgrade without eligibility checks.
 	force bool
-	// versionsURI is a URI pointing to a YAML formatted versions mapping.
-	versionsURI string
+	// showOverrides shows all changed values even if they are specified in inFilename.
```

Comment:
I don't think we need this, it's not useful enough to warrant an extra option. If users are interested in how defaults are changing between versions, they should use profile diff.

Reply:
This option provides an overview of all changes will be applied by upgrade. And the main reason of adding this option is to let users check unexpected changes made even by their CR yaml (mostly by using CLI).

Reply:
Discussed offline, decide to remove it as it is not required for now, and there is an existing tool (profile diff) can do the same job.